frontend/src/utils/assetPath.ts

```diff
@@ -1,5 +0,0 @@
-import { environment } from "src/constants/environments";
```

Comment:
This no longer worked, haven't researched why it was necessary. `import logoPath from "public/image/logo.png"` does.